M2/Macaulay2/packages/Brackets.m2

```diff
@@ -11,7 +11,8 @@ newPackage(
 	      },
 	  PackageImports => {},
           AuxiliaryFiles => false,
-          DebuggingMode => true
+          DebuggingMode => false,
+	  Keywords => "classical invariant theory and Grassmann-Cayley algebras"
```

Comment:
The value of the `Keywords` option should be a *list* of strings.  Ideally, the strings are chosen from the existing headings  from the [packages provided with Macaulay2](https://www.macaulay2.com/doc/Macaulay2/share/doc/Macaulay2/Macaulay2Doc/html/_packages_spprovided_spwith_sp__Macaulay2.html) documentation page